.circleci/config.yml

```diff
@@ -0,0 +1,28 @@
+version: 2.1
```

Comment:
FYI I'm going to be trying this out for App in https://github.com/interexchange/app/pull/4347 as it should cut down on a lot of config we do.